### PR TITLE
Fix/about

### PR DESCRIPTION
**Pull request by marjan-ara**

### CODE REVIEW

File: src/Nowruz/modules/layout/components/footer/footer.tsx [hunk 1]
```diff
@@ -18,17 +19,17 @@ export const Footer: React.FC<FooterProps> = ({ open, logout }) => {
   const userIsLoggedIn = !!currentIdentity;
 
   return (
-    <div className="flex flex-col  items-center w-full h-fit px-4 pb-6 gap-2 mb-0 mt-auto">
+    <MenuList className="flex flex-col  items-center w-full h-fit px-4 pb-6 gap-2 mb-0 mt-auto">
       {userIsLoggedIn && (
         <>
-          <LinkItem
+          {/* <LinkItem
```
Comment (ImanRezasefat) [inline src/Nowruz/modules/layout/components/footer/footer.tsx:25]: remove if not neccessary

File: src/Nowruz/modules/userProfile/containers/createUpdateCertificate/useCreateUpdateCertificate.tsx [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+import { removedEmptyProps } from 'src/core/utils';
+import { Avatar } from 'src/Nowruz/modules/general/components/avatar/avatar';
+import { RootState } from 'src/store';
+import { setUser } from 'src/store/reducers/profile.reducer';
+import * as yup from 'yup';
+
+const schema = yup
+  .object()
+  .shape({
+    name: yup.string().required('Required'),
```
Comment (ImanRezasefat) [inline src/Nowruz/modules/userProfile/containers/createUpdateCertificate/useCreateUpdateCertificate.tsx:18]: I think the message should be more meaningful.

File: src/Nowruz/modules/userProfile/containers/createUpdateExperience/useCreateUpdateExperience.tsx [hunk 3]
```diff
@@ -28,13 +29,13 @@ const schema = yup
...
-    jobCategory: yup.string().required('Required'),
+    jobCategory: yup.string(),
     orgName: yup.string().required('Required'),
     orgId: yup.string(),
-    city: yup.string().required('Required'),
+    city: yup.string(),
     country: yup.string(),
-    employmentType: yup.string(),
-    startMonth: yup.string().required('Required'),
+    employmentType: yup.string().required('Required'),
```
Comment (ImanRezasefat) [inline src/Nowruz/modules/userProfile/containers/createUpdateExperience/useCreateUpdateExperience.tsx:37]: Same here

File: src/Nowruz/modules/layout/components/footer/footer.tsx (hunk 1 above)
Comment (marjan-ara) [reply src/Nowruz/modules/layout/components/footer/footer.tsx:25]: no it might be needed soon

**Review by ImanRezasefat — APPROVED** [review]

**Pull request merged**